glam/api/views.py

```diff
@@ -277,6 +285,28 @@ def _get_fenix_counts(app_id, versions, ping_type, os, by_build):
 
     return data
 
+def _get_fog_counts(app_id, versions, ping_type, os, by_build):
```

Comment:
any reason why you decided to use this separate method instead of changing `_get_fenix_counts()` to `_get_clean_counts()`?

Reply:
To maintain the modularity, in case the definition of the upcoming glean apps is different, we will be able to accomodate. Just as we have separate models for Fenix and Fog